src/components/ProductDetailCard/ProductDetailCard.module.scss

```diff
@@ -0,0 +1,264 @@
+@import '../../styles/variables.scss';
+
+.item {
+  display: flex;
+
+  &__imageBlock {
+    width: 50%;
+    padding-right: 45px;
+  }
+
+  &__stars {
+    color: $cardTitle !important;
+  }
+
+  &__imgWraper {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    gap: 2rem;
+
+    & img {
+      width: 46%;
+      max-height: 880px;
+      object-fit: fill;
+      border-radius: 12px;
+    }
+
+    & img:first-child {
+      width: 100%;
+      object-fit: contain;
+      max-height: 410px;
+    }
+
+    &_pressSide {
+      display: flex;
+      justify-content: space-between;
+
+      & img {
+        width: 46% !important;
+      }
+    }
+  }
+
+  &__discount {
+    width: 100%;
+  }
+
+  &__discount {
+    display: flex;
+    gap: 1em;
+
+    & span:first-child {
+      margin-left: 1em !important;
+    }
+
+    & span {
+      font-family: 'Poppins';
+      font-style: normal;
+      font-weight: 600;
+      font-size: 12px;
+      line-height: 18px;
+      color: $cardButton;
+      background: $amountOfCategory;
+      padding: 2px 8px 2px 8px;
+      border-radius: 12px;
+      text-align: center;
+    }
+  }
+
+  &__body {
+    width: 50%;
+  }
+
+  &__title {
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 48px;
+    color: $cardTitle;
+  }
+
+  &__starBlock {
+    display: flex;
+    align-items: center;
+
+    & p {
+      font-family: 'Open Sans';
```

Comment:
'Open Sans' and 'Poppins' also can be a variable, coz you are using them all time

Reply:
Yes, I want to fix it all together with mixins at refactoring, or at the weekend, since there is a lot to be replaced in many files